.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - iojs-v2.0.0
-  - iojs
+  - "4.0"
+  - "4.1"
```

Comment:
4.2 and "node" or whatever latest is (I could swear there was a better syntax for it. Maybe "stable"?)